clickhouse/base/wire_format.cpp

```diff
@@ -99,4 +100,83 @@ bool WireFormat::SkipString(InputStream& input) {
     return false;
 }
 
+const char quoted_chars[] = {'\0', '\b', '\t', '\n', '\'', '\\'};
+
+inline const char* find_quoted_chars(const char* start, const char* end) {
+    while (start < end) {
```

Comment:
Why not [`std::find_first_of`](https://en.cppreference.com/w/cpp/algorithm/find_first_of) ?

Reply:
it more comfortable for me.
I think [std::find_first_of](https://en.cppreference.com/w/cpp/algorithm/find_first_of) needs
on extra compare because it return 'end' on failure
Or after call we needs compare with 'end' that worse then comparing with 0

Reply:
Please use functions from standard library whenever possible, instead of inventing new ones with 99% overlapping functionality. "comfortable for me" is not a valid excuse...

Reply:
Wath about  extra compare instruction and slow down the solution?

Reply:
First, it was introduced to be used in a very hot piece of code, which I believe is not the case here.
Second, there is a very high chance that compiler can optimize call `std::find_first_of` (i.e. using SSE) way beyond anything you are trying to gain here.
And last, `find_first_symbols_sse2` supports two modes: "return `nullptr`" and "return `end`" [😉](https://github.com/clickHouse/clickhouse/commit/e22eee80558162bea9a0c7cc5f439f45549dc073).

This looks like you are trying to **prematurely** optimize `O(N + 1)` into `O(N)`, by introducing something that would unnecessary complicate the support.